source/OutfitterPanel.cpp

```diff
@@ -323,7 +323,7 @@ int OutfitterPanel::DrawDetails(const Point &center)
 	}
 
 	// Draw this string representing the selected item (if any), centered in the details side panel
-	Point selectedPoint(center.X() - .5 * INFOBAR_WIDTH, center.Y());
+	Point selectedPoint(center.X() - INFOBAR_WIDTH / 2 + 10, center.Y());
```

Comment:
```suggestion
	Point selectedPoint(center.X() - .5 * INFOBAR_WIDTH + 10, center.Y());
```

Might want to stick with the .5*, though.  Just to avoid rounding, in case INFOBAR_WIDTH is odd.

Reply:
Rounding would be beneficial, since drawing things mid-pixel can result in a blur, as shown in #8481.